File: Server/AiesecAppBackEnd/api/models/Project.js
/**
 * Project.js
 *
 * @description :: TODO: You might write a short summary of how this model works and what it represents here.
 * @docs        :: http://sailsjs.org/documentation/concepts/models-and-orm/models
 */

module.exports = {

  schema: true,
  autoCreatedAt: false,
  autoUpdatedAt: false,
  tableName: 'project',
  identity: 'Project',

  attributes: {
    idProject: {
      type: 'integer',
      primaryKey: true,
      unique: true,
      required: true
    },

    LocalCommitte_idLocalCommitte: {
      primaryKey: true,
      model: 'LocalCommitte',
      required: true
    },

    projectName: {
      type: 'string',
      required: true
    },

    version: {
      type: 'string',
      required: true
    },

    shortDescription: {
      type: 'string',
      required: true
    },

    longDescription: {
      type: 'string',
    },

    startDate: {
      type: 'date',
      required: true
    },

    endDate: {
      type: 'date',
      required: true
    },

    createdDate: {
      type: 'datetime'
    },

    expired: {
      type: 'boolean',
      required: true,
      defaultsTo: false
    }
  }
};

File: Client/Aiesec-App/Aiesec_App/Models/Project.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Aiesec_App.Models
{
    public class Project : BaseDataObject
    {
        public int idProject { get; set; }

        public string projectName { get; set; }

        public string version { get; set; }

        public string shortDescritpion { get; set; }

        public string longDescription { get; set; }

        public string startDate { get; set; }

        public string endDate { get; set; }

        public int LocalCommitte_idLocalCommitte { get; set; }
    }
}
